tests/FrameCameraTests.cpp

```diff
@@ -124,20 +147,51 @@ TEST_F(FrameSensorModel, OffBody4) {
 }
 
 
+TEST_F(FrameSensorModel, setFocalPlane1) {
+  int precision  = 20;
+
+   csm::Isd localIsd;
+   loadIsd("data/simpleFramerISD.json",localIsd);
+   csm::ImageCoord imagePt(7.0, 7.0);
+   double ux,uy;
+   vector<double> odtx{0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0};
+   vector<double> odty{0.0,1.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0};
```

Comment:
All 0 vectors should be no distortion. This should just change one coefficient for x and one coefficient for y. Maybe just one coefficient at a time. This thing can be super parameterized.